deploy/052_deploy_donation_box.ts

```diff
@@ -0,0 +1,19 @@
+import { DeployFunction } from "hardhat-deploy/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployer } = await hre.getNamedAccounts();
+
+  // @note if deploying this contract on a chain like Linea that only supports up to
+  // solc 0.8.19, the hardhat.config solc version needs to be overridden and this
```

Comment:
This comment is outdated - Linea supports later versions of solidity but the evm version must be set to london. See https://docs.linea.build/developers/quickstart/ethereum-differences#evm-opcodes

Reply:
I didn't know this. Nice. https://github.com/across-protocol/contracts/pull/690/commits/8e2018374154198f9fdcc56bef5416af7eaf6992